packages/smart-contracts/src/lib/artifacts/BatchPayments/index.ts

```diff
@@ -17,6 +17,11 @@ export const batchPaymentsArtifact = new ContractArtifact<BatchPayments>(
           address: '0xfeE3f8937Ca62DA486a4E067Ed11aA6Bc6B21b4B',
           creationBlockNumber: 10732960,
         },
+        // FIXME: add goerli version
+        goerli: {
+          address: '0x0DD57FFe83a53bCbd657e234B16A3e74fEDb8fBA',
```

Comment:
this contract is not verified! haven't checked others but we should verify all contracts